globals-staging.nix

```diff
@@ -8,7 +8,7 @@ pkgs: {
 
   environmentName = "staging";
 
-  explorerAliases = [];
+  explorerAliases = [ "cardano-explorer.awstest.iohkdev.io" ];
```

Comment:
This was a quick hack and can't be committed. We should probably add explorer aliases to globals.

Reply:
which globals are you talking about?